ProjFS.Linux/PrjFSLib.Linux.Managed/VirtualizationInstance.cs

```diff
@@ -11,13 +12,18 @@ public class VirtualizationInstance
     {
         public const int PlaceholderIdLength = 128;
 
+        private static readonly TimeSpan MountWaitTick = TimeSpan.FromSeconds(0.2);
+        private static readonly TimeSpan MountWaitTotal = TimeSpan.FromSeconds(30);
+
         private ProjFS projfs;
         private int currentProcessId = Process.GetCurrentProcess().Id;
 
         // We must hold a reference to the delegates to prevent garbage collection
+#pragma warning disable IDE0052 // Remove unread private members
```

Comment:
Are these StyleCop pragmas required for a particular build framework?  I don't get warnings here, and the parallel [Mac code](https://github.com/microsoft/VFSForGit/blob/f6c45002f739e3fa6788a2e83ac1824a34ba91b5/ProjFS.Mac/PrjFSLib.Mac.Managed/VirtualizationInstance.cs#L11) doesn't have them, but I assume you've bumped into some kind of StyleCop warning?

If so, would it be appropriate to make these changes in a separate PR, and perhaps include matching ones for the Mac code?

Reply:
They're for the IDE, which otherwise wants to helpfully suggest we delete these fields entirely because they're only written to and never read. But I note the codebase contains no other pragmas of this kind, so maybe the convention is to not use them.